src/amo/components/ReportAbuseButton/index.js

```diff
@@ -18,24 +19,30 @@ import {
 } from 'core/reducers/abuse';
 import { sanitizeHTML } from 'core/utils';
 import Button from 'ui/components/Button';
+import type { AddonType } from 'core/types/addons';
+import type { I18nType } from 'core/types/i18n';
 
 import './styles.scss';
 
 
-type PropTypes = {
+type Props = {
```

Comment:
It looks like this can be an exact type now. I think a bug was preventing it before.

Reply:
heh I thought I tracked all of them, thanks a bunch!